bin/verilator

```diff
@@ -529,6 +531,10 @@ dependency (.d) file is created, this filename will become a source
 dependency, such that a change in this binary will have make rebuild the
 output files.
 
+=item --build
+
+Verilator compiles generated C++ code right after verilation.
```

Comment:
After generating the SystemC/C++ code, Verilator will use the make system to build a library or executable. See also C<--make> and C<--exe>.